public/locales/fr/translation.json

```diff
@@ -142,25 +142,35 @@
         "burn": "Détruire",
         "burn_description": "Le secret sera définitivement supprimé après avoir été lu",
         "burn_after_reading": "Détruire après lecture",
-        "burn_after_time": "Détruire après l’expiration du temps imparti",
+        "burn_after_time": "Détruire après l'expiration du temps imparti",
         "title": "Hemmelig.app",
-        "description": "Partagez des secrets en toute sécurité avec des messages chiffrés qui s’autodétruisent automatiquement après avoir été lus.",
+        "description": "Partagez des secrets en toute sécurité avec des messages chiffrés qui s'autodétruisent automatiquement après avoir été lus.",
         "content": "Contenu",
         "content_description": "Entrez le message secret que vous souhaitez partager en toute sécurité",
         "security": "Sécurité",
         "security_description": "Configurer les paramètres de sécurité de votre secret",
         "private": "Privé",
         "public": "Public",
         "password": "Protection par mot de passe",
-        "secret_description": "Partagez cette URL avec la personne qui a besoin d’accéder au secret",
-        "one_time_use": "Ce secret ne peut être vu qu’une seule fois et s’autodétruira après avoir été lu.",
+        "secret_description": "Partagez cette URL avec la personne qui a besoin d'accéder au secret",
+        "one_time_use": "Ce secret ne peut être vu qu'une seule fois et s'autodétruira après avoir été lu.",
         "norwegian_meaning": "signifie « secret » en norvégien",
         "copy": "Copier dans le presse-papiers",
         "copied": "Copié !",
         "create": "Créer",
         "sign_in": "Connexion",
         "complete_url": "URL secrète complète",
-        "complete_url_description": "Il s’agit de l’URL complète qui inclut la clé de déchiffrement. Partagez cette URL pour donner un accès direct au secret."
+        "complete_url_description": "Il s'agit de l'URL complète qui inclut la clé de déchiffrement. Partagez cette URL pour donner un accès direct au secret.",
+        "password_generator": "Générateur de mot de passe",
+        "password_generator_description": "Générer un mot de passe sécurisé avec des exigences personnalisées",
```

Comment:
Those two are not present in the English translation file.

Reply:
ah, yes, this feature was deleted